apisix/plugins/hmac-auth.lua

```diff
@@ -79,7 +82,17 @@ local consumer_schema = {
             type = "boolean",
             title = "Whether to escape the uri parameter",
             default = true,
-        }
+        },
+        validate_request_body = {
+            type = "boolean",
+            title = "A boolean value telling the plugin to enable body validation.",
+            default = false,
+        },
+        max_req_body = {
+            type = "integer",
+            title = "Max request body allowed",
```

Comment:
"Max request body size" would be better.

Reply:
OK